server/data/contacts.ts

```diff
@@ -0,0 +1,314 @@
+import { cycles, users } from '../config/mongoCollections';
+import { ObjectId } from 'mongodb';
+import { checkObjectId, checkNonEmptyString, checkEmail, checkName } from '../helpers';
+import { ContactObject } from '../typings';
+
+
+/**
+ * Get contact by id
+ * @param {string} userId
+ * @param {string} applicationId
+ * @param {string} contactId
+ * @returns {Promise<Object>} The contact given the ids above
+ */
+export const getContactById = async (userId: string, applicationId: string, contactId: string):Promise<Object> => {
+
+    if(checkObjectId(userId)){
```

Comment:
checkObjectId returns true if the id is valid, so you want to error when it's false instead. This goes for other error checks as well...